addons/sale_timesheet/static/tests/mock_server/mock_models/sale_order_line.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class SaleOrderLine extends models.ServerModel {
+    _name = "sale.order.line";
```

Comment:
models should be defined in their main module otherwise we might run into some strange dependency issues if current module is not installed